tox.ini

```diff
@@ -75,5 +75,6 @@ commands =
     bottleneck: pip install git+https://github.com/pydata/bottleneck.git@master#egg=bottleneck
     lm3: pip install git+https://github.com/OpenHydrology/lmoments3.git@develop#egg=lmoments3
     doctest: pytest --rootdir tests/ --xdoctest xclim
-    pytest --cov xclim --basetemp={envtmpdir}
+    pytest --cov xclim --basetemp={envtmpdir} -m "not slow"
+    slow: pytest --cov xclim --basetemp={envtmpdir} -m "slow"
```

Comment:
Nice, but will this run the "not slow" followed by "slow" tests? Just want to make sure we don't run the same tests twice (that would be _very_ slow).

Reply:
Yes! This should run all tests only once in the `py36-slow` recipe, and all others recipes would run only the tests NOT marked as slow.

Reply:
Wicked. That's simple enough!

Reply:
Oh and I realize I'm not sure how coverage is computed here? How do we ensure all tests are counted?

Reply:
Coverage is based on reports from multiple builds. I'll need to double-check but I believe the results from both pytest calls are saved in a SQL DB before we send them over to `coveralls`.